google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/flexible/AppEngineFlexibleSupportProvider.java

```diff
@@ -165,6 +128,48 @@ public static void addSupport(@NotNull AppEngineFlexibleFacet facet,
     setupDeploymentRunConfiguration(facet.getModule());
   }
 
+  /**
+   * Generates an app.yaml configuration file in the src/main/appengine directory. If an app.yaml
+   * already exists it will not overwrite the file.
+   */
+  private static void generateAppYaml(@NotNull AppEngineFlexibleFacet facet,
+      @NotNull ModifiableRootModel rootModel) {
+    Project project = facet.getModule().getProject();
+    FileTemplate appYamlTemplate = FileTemplateManager.getInstance(project)
+        .getInternalTemplate(AppEngineTemplateGroupDescriptorFactory.APP_YAML_TEMPLATE);
+
+    try {
+      VirtualFile virtualFile = rootModel.getContentRoots()[0].findFileByRelativePath("src/main");
```

Comment:
I don't know. I think it's possible to delete every content root from a project/module. Can't hurt to guard?